### ISSUE TITLE
Google Gantt Chart - chart.setselection() request.

### ISSUE THREAD

**Issue by adabadgh** [google/google-visualization-issues#2119]
Raising a request to add chart.setselection() functionality to the google gantt chart.

**Comment by heennkkee** [comment]
Looks at the documentation like it already exists?
https://developers.google.com/chart/interactive/docs/gallery/ganttchart#methods

**Comment by grabks** [comment]
@heennkkee That part of the documentation page is the same for all charts. setSelection() is actually currently broken for Gantt (and the behavior of getSelection() is a bit surprising as well.)

**Comment by heennkkee** [comment]
That's true @grabks , I didn't even check if it was working, embarrasing enough.

Looks like the selection doesn't pass any data, just the event..

**Comment by grabks** [comment]
Not 100% sure what you mean, @heennkkee. The `'select'` event never passes any relevant data. For any chart, we expect you to call `chart.getSelection()` when you get the `'select'` event in order to get the selection. The issue with Gantt's `getSelection` is that it appears to return an array of empty objects, instead of them having some sort of identifier of what they are, like a row.

**Comment by heennkkee** [comment]
Hah, that's what I meant, of course, I should really start reading through my own post more carefully... My bad :) @grabks

**Comment by crutchcorn** [comment]
Is there any workaround for this? It seems as if this issue is still persisting today
